wca-guidelines.md

```diff
@@ -161,6 +161,7 @@ To be more informative, each Guideline is classified using one of the following
 - A3c4++) [CLARIFICATION] The competitor may remove the object during the solve, if they did not do it during the inspection (e.g. they forgot).
 - A5b+) [CLARIFICATION] While inspecting or solving the puzzle, the competitor may touch the puzzle with any part of their body. Exception: One-Handed Solving and Solving With Feet (see [Regulation C1b](regulations:regulation:C1b) and [Regulation D1b](regulations:regulation:D1b)).
 - A6b+) [EXPLANATION] The arbitrary value of 0.06 seconds was chosen to accommodate concerns about Stackmat timer malfunctions.
+- A6e+) [CLARIFICATION] If the puzzle falls in the lap of the competitor after stopping the timer, this is considered as touching the puzzle.
```

Comment:
"falls in", or "falls on"?

Reply:
"falls into" sounds most natural to me, although "falls in" sounds fine, too.

Reply:
Updated to the most natural sounding version. ;-)